shenyu-plugin/shenyu-plugin-motan/src/main/java/org/apache/shenyu/plugin/motan/cache/ApplicationConfigCache.java

```diff
@@ -185,7 +186,7 @@ public RefererConfig<CommonHandler> build(final MetaData metaData) {
         });
         reference.setGroup(motanParamExtInfo.getGroup());
         reference.setVersion("1.0");
-        reference.setRequestTimeout(1000);
+        reference.setRequestTimeout(motanParamExtInfo.getTimeout());
```

Comment:
If client is not upgraded, `motanParamExtInfo.getTimeout()` will be `null`, so you should use `Optional.ofNullable(motanParamExtInfo.getTimeout()).orElse(1000)`